src/plugins/console/public/services/autocomplete.ts

```diff
@@ -72,8 +73,16 @@ export class AutocompleteInfo {
     }
   }
 
+  /**
+   * Indicates if autocomplete_entities fetching is in progress.
+   */
+  private readonly _isLoading$ = new BehaviorSubject<boolean>(false);
+  public readonly isLoading$ = this._isLoading$.asObservable();
```

Comment:
Do we use this second variable somewhere?

Reply:
Yeah we actually need both of them:
The first line creates a private BehaviorSubject to track a boolean loading state, and the second exposes the loading state as a public observable so that we can subscribe to that from somewhere else in the app (in our case in the editor).

Reply:
I understand, thanks!